redisinsight/ui/src/pages/browser/modules/key-details/components/rejson-details/RejsonDetailsWrapper.tsx

```diff
@@ -80,42 +74,33 @@ const RejsonDetailsWrapper = (props: Props) => {
       />
       <div className="key-details-body" key="key-details-body">
         {!loading && (
-        <div className="flex-column" style={{ flex: '1', height: '100%' }}>
-          <div
-            data-testid="json-details"
-            className={`${[styles.container].join(' ')}`}
-          >
-            {loading && (
-              <EuiProgress
-                color="primary"
-                size="xs"
-                position="absolute"
-                data-testid="progress-key-json"
-              />
-            )}
-            {!(loading && data === undefined) && (
-              <RejsonDetails
-                selectedKey={selectedKey}
-                dbNumber={0}
-                dataType={type || ''}
-                deleteMsg=""
-                instanceId={123}
-                resultTableKeyMap={{}}
-                handleSubmitJsonUpdateValue={handleSubmitJsonUpdateValue}
-                onJSONPropertyDeleted={reportJSONPropertyDeleted}
-                data={data}
-                onJSONKeyExpandAndCollapse={reportJSONKeyExpandAndCollapse}
-                onJSONPropertyAdded={reportJSONPropertyAdded}
-                onJSONPropertyEdited={reportJSONPropertyEdited}
-                shouldRejsonDataBeDownloaded={!downloaded}
-                handleSubmitUpdateValue={handleEditValueUpdate}
-                handleDeleteKeyDialogOpen={() => {}}
-                handleOpenExpiryDialog={() => {}}
-                keyProperty={{}}
-              />
-            )}
+          <div className="flex-column" style={{ flex: '1', height: '100%' }}>
+            <div
+              data-testid="json-details"
+              className={`${[styles.container].join(' ')}`}
```

Comment:
instead of join you should use cx from classnames lib, but here we have only one class
please, review all other places